server/portal/settings/settings.py

```diff
@@ -615,8 +615,8 @@ def gettext(s): return s  # noqa:E731
 
 PORTAL_KEY_SERVICE_ACTOR_ID = getattr(settings_custom, '_PORTAL_KEY_SERVICE_ACTOR_ID', "jzQP0EeX7mE1K")
 
-PORTAL_JOB_NOTIFICATION_STATES = ["PENDING", "STAGING_INPUTS", "SUBMITTING", "QUEUED", "RUNNING",
-                                  "CLEANING_UP", "FINISHED", "STOPPED", "FAILED", "BLOCKED", "PAUSED"]
+PORTAL_JOB_NOTIFICATION_STATES = ["PENDING", "PROCESSING_INPUTS", "STAGING_INPUTS", "STAGING_JOB", "SUBMITTING_JOB", "QUEUED", "RUNNING",
+                                  "ARCHIVING", "BLOCKED", "PAUSED", "FINISHED", "CANCELLED", "FAILED"]
```

Comment:
@shayanaijaz Thanks for updating these. I think, from CMD last time, we can exclude `PROCESSING_INPUTS`, but could you confirm the desired notification states with them in #cep-design?

Reply:
Given the [updates](https://confluence.tacc.utexas.edu/display/UP/WIP%3A+UI+-+Constants+-+Job+Status) from CMD, we should choose only one status per translation state, to prevent duplicate translated events. e.g. instead of having 4 notifications for `Queueing`, choose only one (perhaps the first?) from those states

Reply:
In the frontend there is a check to make sure multiple toasts don't popup for the same "text" value of a status. I can add something in the backend too so we are not sending unnecessary notifications.